config/manager/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/kubestellar/kubestellar/controller-manager
-  newTag: 0.20.0-alpha.1
+  newTag: 0.20.0-rc1
```

Comment:
Shouldn't this be a placeholder?

Reply:
ok, done. Used `will-be-replaced` here as well

Reply:
using `will-be-replaced` here and in the chart broke the e2e tests, so I reverted to `0.20.0-alpha.1`. This is going to be addressed by #1779 so I am not trying to solve this in this PR.